src/Home_Work/Lesson_2.java

```diff
@@ -0,0 +1,58 @@
+
+package Home_Work;
+
+public class Lesson_2 {
+    public static void main(String[] args) {
+        IntSum(12, 4);
```

Comment:
методы именуются с прописной буквы
```suggestion
        intSum(12, 4);
```